src/main/resources/lang/default.lang

```diff
@@ -1939,6 +1939,19 @@ quit reasons:
 	kicked: kicked
 	timed_out: timed out
 
+# -- Transform Reasons --
+transform reasons:
+	cured: zombie cure, zombie curing, zombie cured, cured
+	drowned: entity drowned, drowned, zombie converting to drown, zombie drown
+	frozen: entity froze, entity frozen, skeleton frozen, skeleton froze, skeleton converting to stray
+	lightning: lightning, creeper super charge, super charged creeper
+	metamorphosis: tadpole convert, tadpole converted to frog, tadpole metamorphosis, metamorphosis
+	piglin_zombified: piglin zombification, piglin zombified
+	sheared: sheared, mooshroom sheared
+	split: slime split, slime splitting, split, magma slime split, magma slime splitting
+	unknown: unknown
+	infection: infection
```

Comment:
For some reason my other review lost my comments here:
These are way too loose when combined with the event pattern:
* `on unknown` is just bad
* `on lightning` overwrites the existing lightning event
* `on sheared` is misleading, I'd think it should work on sheep given the name
* `on cured`, `on infection`, `on metamorphosis`, and `on split` are ok but very lacking in context
* `on drowned` can easily be misleading

Basically these names should be much stricter or the event's pattern needs to change. Perhaps something like `on transform due to %*transformreason%`